conf/variant/intel/local.conf.sample

```diff
@@ -6,6 +6,4 @@ EFI_PROVIDER = "grub-efi"
 
 WKS_FILE = "mkefidisk-multiboot.wks"
 
-IMAGE_INSTALL_append = "\
-    grub-efi \
```

Comment:
This is very strange as I can see several spaces before grub-efi. Anyway if it fixes the issue then I am fine with it.